breacharbiter.go

```diff
@@ -541,7 +709,8 @@ func (b *breachArbiter) breachObserver(contract *lnwallet.LightningChannel,
 			desc.SigHashes = hc
 			desc.InputIndex = inputIndex
 
-			return lnwallet.CommitSpendNoDelay(b.wallet.Cfg.Signer, &desc, tx)
+			return lnwallet.CommitSpendNoDelay(
+				b.wallet.Cfg.Signer, &desc, tx)
```

Comment:
I have resolved and simplified the logic for doing so in my future PR for sweeping second layer HTLCs. Talked w/roasbeef and we have decided to leave as is and apply the change in the subsequent PR.